data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -258,66 +261,99 @@
 	 header or encapsulation technology e.g. <xref target="dn_svcs"/> or it
 	 may be achieved by other standardized technologies e.g.  <xref
 		 target="dn_svc_encaps"/>.
-	
          </t>
          <t>
 	 Extensibility of that Data Plane:  Whether or not the DetNet data plane
 	 includes the facility to carry additional information (metadata) that
 	 can be used to provide an enhanced service to the DetNet packet.
+          </t>
+        </list>
+         </t>
+         <t>
+          (Below replacement text suggested by Balazs)
+         </t>
+         <t>
+         <list style='numbers'>
+         <t>
+          Forwarding paradigm: DetNet can be built over packet switched (IP)
+          networks or label switched (MPLS) networks .  

```

Comment:
how about: s/can be built over/is currently defined for operation

Reply:
Fixed